### ISSUE TITLE
one sentence per line

### ISSUE THREAD

**Issue by Kaligule** [purcell/unfill#7]
I have used `unfill-toggle` for a long time, thanks a lot.
Lately I had to write a lot of plain text and came up with an improvement: Cycling through 3 states instead of 2.
Especially I wanted a function that formats  the current paragraph to have one line per sentence.

I blogged about it [here](https://schauderbasis.de/posts/reformat_paragraph/). Would you be interested in adopting a function like this into your unfill package? I would be honored to help doing so.

**Comment by purcell** [comment]
Yes, potentially! It could be interesting to have `unfill-toggle-functions` be a variable containing `(fill-paragraph unfill-paragraph)` and then also provide an `unfill-fill-sentences-in-paragraph` function that users could optionally insert into that variable to enable your extended functionality in certain modes.

**Comment by Kaligule** [comment]
Wow, that is a cool idea on how to make that setup an extension on the current package.

**Comment by purcell** [comment]
There are a few patterns like that which you'll probably encounter in due course. I've just seen more at this stage! 😄
